receiver/prometheusreceiver/internal/metricsbuilder.go

```diff
@@ -29,17 +29,25 @@ import (
 	"github.com/prometheus/prometheus/pkg/textparse"
 )
 
-const metricsSuffixCount = "_count"
-const metricsSuffixBucket = "_bucket"
-const metricsSuffixSum = "_sum"
-const startTimeMetricName = "process_start_time_seconds"
+const (
+	metricsSuffixCount      = "_count"
+	metricsSuffixBucket     = "_bucket"
+	metricsSuffixSum        = "_sum"
+	startTimeMetricName     = "process_start_time_seconds"
+	scrapeLatencyMetricName = "scrape_duration_seconds"
+	scrapeStatusMetricName  = "up"
+	scrapeStatusOk          = "200"
+	scrapeStatusErr         = "404"
```

Comment:
Not sure if 404 is the best to describe the error. Just need to document this.

Reply:
Done.